.github/workflows/go.yml

```
@@ -0,0 +1,22 @@
+name: Go
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.20'
+
+    - name: Test
+      run: go test -v .
```

Maybe already run ...

```suggestion
      run: go test -v ./...
```

?

---

That will run tests in all the subdirectories as well but we don't have any tests there so it just outputs 'no tests found'